src/imxrt1180/lpuart.rs

```diff
@@ -0,0 +1,174 @@
+use super::pads::{
+    gpio_ad::*, gpio_aon::*, gpio_b1::*, gpio_b2::*, gpio_emc_b1::*, gpio_emc_b2::*, gpio_sd_b2::*,
+};
+use crate::{
+    consts::*,
+    lpuart::{Pin, Rx, Tx},
+    Daisy,
+};
+
+uart!(module: U1, alt: 0b0000, pad: GPIO_AON_08, direction: Tx, daisy: None);
+uart!(module: U1, alt: 0b0000, pad: GPIO_AON_09, direction: Rx, daisy: None);
+uart!(module: U2, alt: 0b0110, pad: GPIO_AON_01, direction: Rx, daisy: Some(DAISY_LPUART2_IPP_IND_LPUART_RXD_SELECT_GPIO_AON_01));
+uart!(module: U2, alt: 0b0010, pad: GPIO_AON_27, direction: Rx, daisy: Some(DAISY_LPUART2_IPP_IND_LPUART_RXD_SELECT_GPIO_AON_27));
+uart!(module: U2, alt: 0b0110, pad: GPIO_AON_00, direction: Tx, daisy: Some(DAISY_LPUART2_IPP_IND_LPUART_TXD_SELECT_GPIO_AON_00));
+uart!(module: U2, alt: 0b0010, pad: GPIO_AON_26, direction: Tx, daisy: Some(DAISY_LPUART2_IPP_IND_LPUART_TXD_SELECT_GPIO_AON_26));
+uart!(module: U3, alt: 0b0011, pad: GPIO_EMC_B1_02, direction: Rx, daisy: Some(DAISY_LPUART3_IPP_IND_LPUART_RXD_SELECT_GPIO_EMC_B1_02));
+uart!(module: U3, alt: 0b0110, pad: GPIO_AD_14, direction: Rx, daisy: Some(DAISY_LPUART3_IPP_IND_LPUART_RXD_SELECT_GPIO_AD_14));
+uart!(module: U3, alt: 0b0011, pad: GPIO_EMC_B1_03, direction: Tx, daisy: Some(DAISY_LPUART3_IPP_IND_LPUART_TXD_SELECT_GPIO_EMC_B1_03));
+uart!(module: U3, alt: 0b0110, pad: GPIO_AD_13, direction: Tx, daisy: Some(DAISY_LPUART3_IPP_IND_LPUART_TXD_SELECT_GPIO_AD_13));
+uart!(module: U5, alt: 0b0011, pad: GPIO_EMC_B1_15, direction: Rx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_RXD_SELECT_GPIO_EMC_B1_15));
+uart!(module: U5, alt: 0b0010, pad: GPIO_EMC_B1_36, direction: Rx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_RXD_SELECT_GPIO_EMC_B1_36));
+uart!(module: U5, alt: 0b0010, pad: GPIO_EMC_B2_18, direction: Rx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_RXD_SELECT_GPIO_EMC_B2_18));
+uart!(module: U5, alt: 0b0001, pad: GPIO_AD_27, direction: Rx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_RXD_SELECT_GPIO_AD_27));
+uart!(module: U5, alt: 0b0110, pad: GPIO_SD_B2_09, direction: Rx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_RXD_SELECT_GPIO_SD_B2_09));
+uart!(module: U5, alt: 0b0010, pad: GPIO_EMC_B1_14, direction: Tx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_TXD_SELECT_GPIO_EMC_B1_14));
+uart!(module: U5, alt: 0b0010, pad: GPIO_EMC_B1_35, direction: Tx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_TXD_SELECT_GPIO_EMC_B1_35));
+uart!(module: U5, alt: 0b0010, pad: GPIO_EMC_B2_17, direction: Tx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_TXD_SELECT_GPIO_EMC_B2_17));
+uart!(module: U5, alt: 0b0001, pad: GPIO_AD_26, direction: Tx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_TXD_SELECT_GPIO_AD_26));
+uart!(module: U5, alt: 0b0110, pad: GPIO_SD_B2_08, direction: Tx, daisy: Some(DAISY_LPUART5_IPP_IND_LPUART_TXD_SELECT_GPIO_SD_B2_08));
```

Comment:
`GPIO_SD_B2_08` is missing from the table in chapter 13 of the reference manual. Nevertheless, it supports LPUART5 transmit.

<img width="972" alt="image" src="https://github.com/user-attachments/assets/f6e66c4e-f456-42cb-b902-437418ced887" />